style.css

```diff
@@ -39,7 +39,7 @@ nav {
   background-color: transparent;
 }
 
-.navigationList {
+#navigationList {
```

Comment:
Good job with the background blur, it looks fantastic 👌  But let's make sure when the user clicks on the menu link, the menu closes.
<img width="763" alt="Screenshot 2023-03-06 at 11 43 38 PM" src="https://user-images.githubusercontent.com/25887253/223227559-a4d74b94-77f7-455d-a640-d6af53c75df6.png">
![Screen Recording 2023-03-06 at 11 41 36 PM](https://user-images.githubusercontent.com/25887253/223227601-c899224a-6850-461f-999c-ec8235f1466a.gif)